fetch_data.py

```diff
@@ -56,6 +57,10 @@ def check_imdb(datasets_folder):
         tar.close()
         os.remove(archive_path)
 
+        unsup_directory = os.path.join(imdb_path, 'aclImdb', 'train', 'unsup')
+        if os.path.exists(unsup_directory):
+            shutil.rmtree(unsup_directory)
+
```

Comment:
yes. a good thing might be using ``load_files`` and check that there's exactly two labels in it afterwards.